src/mapgen.cpp

```diff
@@ -1460,12 +1460,14 @@ class jmapgen_computer : public jmapgen_piece
 {
     public:
         std::string name;
+        std::string access_denied;
```

Comment:
```suggestion
        translation access_denied;
```

Reply:
Thanks for your feedback! Why isn't the `name` field also a `translation`? Corresponding strings like "Prototype DARPA-713 AUTHORIZED PILOTS ONLY" still seem to end up in the .po files, so I'm a bit confused about the difference. Is `name` using an older way of working that's being phased out?

Reply:
> Is name using an older way of working that's being phased out?

Yes. the `translation` class offers better type safety and allows specifying translation context in the JSON files.